release_notes_generator/generator.py

```diff
@@ -125,3 +126,35 @@ def generate(self) -> Optional[str]:
         )
 
         return release_notes_builder.build()
+
+    def get_latest_release(self, repo: Repository) -> Optional[GitRelease]:
```

Comment:
I am a bit confused, because I can see this method being added, but I see that originally it was called by
```
rls = self._safe_call(repo.get_latest_release)()
```
so why there is no removal of the original? What am I missing?

Reply:
The new method has the same name as the one on the repo instance.
In the previous code, there were direct calls, but now this call is done using the new method to include tag information.